### PR TITLE
Replace '~' with $HOME to prevent strikethrough on MD render

### PR DESCRIPTION
**Pull request by kbambridge**
**What this PR does / why we need it**:
Minor fixes to markdown formatted documentation

**Special notes for your reviewer**:

**If applicable**:
- [x] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

**Comment by bacongobbler** [comment]
Hey! Thanks so much for the PR.

The documentation you're amending in this PR is auto-generated by calling `make docs` and is based on the `helm help` output from [here](https://github.com/helm/helm/blob/a47a35814973bbaa7d37eec9fa5e30ac8c863daf/cmd/helm/helm.go#L47-L77), which is why your PR is failing on CircleCI. In order to make this change, please

- make the proposed change here https://github.com/helm/helm/blob/a47a35814973bbaa7d37eec9fa5e30ac8c863daf/cmd/helm/helm.go#L47-L77
- once that's done, call `make docs` to re-generate the official documentation with that change
- we'll also need a DCO sign-off. More info here: https://github.com/helm/helm/pull/5128/checks?check_run_id=46685024

Let us know if you need any help with this. Thanks!

File: docs/helm/helm.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ Common actions from this point include:
 - helm list:      list releases of charts
 
 Environment:
```
Comment (hickeyma) [inline docs/helm/helm.md]: I wonder should all the environment variables be put in a bullet point list? At the moment when MD rendered, they are just one big blob of text?

**Comment by hickeyma** [comment]
@k This looks like the MD is now fixed by https://github.com/helm/helm/pull/5644. Can you have a look at the [docs](https://helm.sh/docs/helm/#helm) and if you are happy, maybe we can close this PR?

**Comment by bridgetkromhout** [comment]
Thanks again for your PR. This was solved in https://github.com/helm/helm/pull/5644 so we're closing this PR now, but please do check out Helm 3 (currently in beta) if desired!

**Pull request closed without merging**